File: 36-day/randomNumbers.js
function randomNumbers2(dices, faces) {
    let results = new Array()
    for (let d = 0; d < dices; d++) {
        results.push(Math.ceil(Math.random() * faces))
    }
    return results
}

function throwDices(dices) {
    return randomNumbers2(dices, 6)
}

const numberDices = parseInt(prompt('How many dice are you going to roll? '))
alert(`Result: ${throwDices(numberDices)}`)

File: 36-day/randomNumbers.py
from random import choice

def randomNumbers2(dices, faces):
    results = []
    for d in range(0, dices)

results.append(choice(range(1, faces + 1)))
    return results

def throwDices(dices):
    return randomNumbers2(dices, 6)

numbersGiven = int(input('How many dice are you going to roll? '))
print('Point: {}'.format(throwDices(numbersGiven)))